app/__init__.py

```diff
@@ -94,4 +100,13 @@ def logout():
 @app.route('/')
 def show_homepage():
     """Shows website homepage"""
-    return render_template('homepage.html')
\ No newline at end of file
+    return render_template('homepage.html', testing_states=testing_states)
+
+
+@app.route('/location')
+def show_state_locations():
+    """Return selected state from drop down menu"""
```

Comment:
Since this function shows the testing locations in a state, we can restructure the comment to "Return testing locations for state selected from drop down menu"